### DISCUSSION TITLE
images with ```{image}

### DISCUSSION THREAD

**Discussion by parmentelat** [executablebooks/meta#283] [Q&A] [answered]
I was thrilled to read 
https://jupyterbook.org/content/figures.html#images 
but I'm struggling a bit and could use some additional info

I have tried the myst image syntax, i.e.

      ```{image} ../media/myimage.png
      :width: 200px
      :align: center
      ```
this fragment renders correctly in jb's output, but .. not in the notebook, and I feel like I'm missing something obvious

then conversely, when using a raw html `<img>` tag, I can see it in the notebook and not in the book

what may I be doing wrong ?

**Comment by chrisjsewell** [comment, marked as answer]
Heya, did you add the html_image extension to your config: https://jupyterbook.org/content/figures.html#raw-html-images

**Reply by parmentelat** [reply]
Hi
yes right, thanks ! that was my problem indeed, I was just figuring it out by myself :)

any clue as to why the ```{image}  thingy won't render in the notebook ?

thanks in any case !

**Reply by chrisjsewell** [reply]
No they won’t render directly in the notebook yet I’m afraid

**Reply by parmentelat** [reply]
ah ok, thanks for making this clear

**Comment by melissawm** [comment]
Sorry to bump an old issue - I am also struggling with this. The current situation is
- Using the `image` directive, images show up in the rendered jb output, but are not correctly rendered when opening as a Jupyter notebook
- If the usual markdown syntax is used i.e. `![alt](image.png)`, the image shows up both in the notebook and rendered html, but cannot be resized or aligned.

Is there a way to do this currently, or is this a planned feature? What is the best approach to deal with this problem? 

Thanks!
